src/arm.rs

```diff
@@ -30,6 +30,43 @@ pub unsafe fn __aeabi_uldivmod() {
     intrinsics::unreachable();
 }
 
+#[naked]
+#[cfg_attr(not(test), no_mangle)]
+pub unsafe fn __aeabi_idivmod() {
```

Comment:
Would it make sense to add a test for this assembly routine (and the one below)? Just to check its expected signature (`fn(u32, u32) -> (u32,u32)` in this case, I think) works as expected?